Sources/Testing/ABI/v0/ABIv0.Record+Streaming.swift

```diff
@@ -31,18 +31,15 @@ extension ABIv0.Record {
   ) -> Event.Handler {
     let humanReadableOutputRecorder = Event.HumanReadableOutputRecorder()
     return { event, context in
-      if case let .runStarted(plan) = event.kind {
```

Comment:
This diff produces in no change to the contents of the event stream no non-listing scenarios, right?

Reply:
Correct. When running `swift test`, the JSON output is identical. The actual enumeration happens slightly earlier in the execution flow, but from the POV of the caller, it's the same because the logic inside Swift Testing is responsible for emitting the test objects before the `runStarted` event object in both cases.